tools/precompiler/precompiler.py

```diff
@@ -68,12 +68,12 @@ def _compile(options: "argparse.Namespace") -> None:
 # A stub type alias for readability.
 # See the Bazel WorkRequest object definition:
 # https://github.com/bazelbuild/bazel/blob/master/src/main/protobuf/worker_protocol.proto
-JsonWorkerRequest = object
+JsonWorkRequest = object
```

Comment:
All of the type hints below seem to expect `JsonWorkRequest` not `JsonWorkerRequest`